src/edu/gatech/cs2340/trydent/TrydentEngine.java

```diff
@@ -0,0 +1,145 @@
+package edu.gatech.cs2340.trydent;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Main class that kicks off the TrydentEngine.
+ * 
+ * As a developer using the TrydentEngine, all you have to do is call
+ * TrydentEngine.start() to get started.
```

Comment:
IMO, feel free to leave it for now, but maybe we should discuss this in the future.